### PR TITLE
[Legacy Code] UISearchBar+RACSignalSupport category 

### PR DESCRIPTION
**Pull request by vasilenkoigor**
This PR for #3109 issue
Implemented RACSignalSupport category for UISearchBar

### CODE REVIEW

**Comment by NachoSoto** [comment]
I haven't reviewed this code yet, but the header will need to be imported from `ReactiveCocoa.h`.

**Comment by vasilenkoigor** [comment]
@NachoSoto Fixed 👍

File: ReactiveCocoa/ReactiveCocoa.h [hunk 1]
```diff
@@ -73,7 +73,7 @@ FOUNDATION_EXPORT const unsigned char ReactiveCocoaVersionString[];
 	#import <ReactiveCocoa/UITableViewHeaderFooterView+RACSignalSupport.h>
 	#import <ReactiveCocoa/UITextField+RACSignalSupport.h>
 	#import <ReactiveCocoa/UITextView+RACSignalSupport.h>
-
+    #import <ReactiveCocoa/UISearchBar+RACSignalSupport.h>
```
Comment (mdiep) [inline ReactiveCocoa/ReactiveCocoa.h]: This should be indented to match the other imports and alphabetized. 😄

Comment (vasilenkoigor) [inline ReactiveCocoa/ReactiveCocoa.h]: Oops😀

Comment (vasilenkoigor) [inline ReactiveCocoa/ReactiveCocoa.h]: hhmm..buuut, in Xcode and AppCode my import indented to match the other imports

Comment (mdiep) [inline ReactiveCocoa/ReactiveCocoa.h]: It should be indented with a tab—not 4 spaces. ☺️

Comment (vasilenkoigor) [inline ReactiveCocoa/ReactiveCocoa.h]: Oh, I'm stupid😀
Fixed again :)

Comment (vasilenkoigor) [inline ReactiveCocoa/ReactiveCocoa.h]: 😀😀😀😀

**Comment by mdiep** [comment]
The Objective-C ReactiveCocoa APIs have been moved to https://github.com/ReactiveCocoa/ReactiveObjC/. If you're interested in continuing this PR, please reopen it against that repository. Thanks!

**Pull request closed without merging**